Makefile

```diff
@@ -27,57 +29,68 @@ clean: ## Clean build directory
 	cd $(DOCS_DIR) && rm -rf $(BUILDDIR)/*
 
 .PHONY: build
-build:  ## Set up training: Install requirements
+build:		## Set up training: Install requirements
 	python3 -m venv . || virtualenv --clear --python=python3 .
 	bin/python -m pip install --upgrade pip
 	bin/pip install -r requirements.txt
-	@echo
-	@echo "Please activate your Python virtual environment with"
-	@echo "source bin/activate"
+
+bin/python:
+	python3 -m venv . || virtualenv --clear --python=python3 .; \
+	bin/python -m pip install --upgrade pip; \
+	bin/pip install -r requirements.txt
```

Comment:
This target is the same as `build`, but with statement delimiters. These two targets should be merged into one. I prefer the cleaner syntax of `build` without statement delimiters, and prefer the target name of `bin/python`. Is there a reason for using statement delimiters?

Reply:
Fixed in 3ee76c9f93654dbbe9505b70c6054ccf9ca30797